src/read.c

```diff
@@ -1504,8 +1502,13 @@ static avifResult avifDecoderGenerateImageGridTiles(avifDecoder * decoder, avifI
                                   (const char *)firstTileItem->type);
             return AVIF_RESULT_INVALID_IMAGE_GRID;
         }
+        progressive = progressive && item->progressive;
```

Comment:
Nit/optional: I usualy write this as:

```
        if (!item->progressive) {
            progressive = AVIF_FALSE;
        }
```

But your version makes it clear that we are AND'ing all the `item->progressive` values.

Reply:
Done.